postman/Platform Core API.postman_collection.json

```diff
@@ -2665,6 +2761,554 @@
+                    }
+                  }
+                },
+                "url": {
+                  "raw": "{{baseUrl}}/v1/users/authenticationproviders/:id",
+                  "host": ["{{baseUrl}}"],
+                  "path": ["v1", "users", "authenticationproviders", ":id"],
+                  "variable": [
+                    {
+                      "key": "id",
+                      "value": null
+                    }
+                  ]
+                }
+              },
+              "response": []
+            },
+            {
+              "name": "Delete an Authentication Provider",
+              "request": {
+                "method": "DELETE",
+                "header": [],
+                "body": {
+                  "mode": "raw",
+                  "raw": "",
+                  "options": {
+                    "raw": {
+                      "language": "json"
+                    }
+                  }
+                },
+                "url": {
+                  "raw": "{{baseUrl}}/v1/users/authenticationproviders/:id",
+                  "host": ["{{baseUrl}}"],
+                  "path": ["v1", "users", "authenticationproviders", ":id"],
+                  "variable": [
+                    {
+                      "key": "id",
+                      "value": null
+                    }
+                  ]
+                }
+              },
+              "response": []
+            }
+          ]
+        },
+        {
+          "name": "OpenID Configuration",
+          "item": [
+            {
+              "name": "Update OpenID Configuration",
+              "request": {
+                "method": "PUT",
+                "header": [],
+                "body": {
+                  "mode": "raw",
+                  "raw": "{\n  \"interactionHook\": {\n    \"url\": \"CALLBACK_URL\",\n    \"claims\": [],\n    \"disabled\": false\n  }\n} ",
+                  "options": {
+                    "raw": {
+                      "language": "json"
+                    }
+                  }
+                },
+                "url": {
+                  "raw": "{{baseUrl}}/v1/openid/configuration",
+                  "host": ["{{baseUrl}}"],
+                  "path": ["v1", "openid", "configuration"]
+                }
+              },
+              "response": []
+            },
+            {
+              "name": "Retrieve OpenID configuration",
+              "request": {
+                "method": "GET",
+                "header": [],
+                "url": {
+                  "raw": "{{baseUrl}}/core/v1/openid/configuration",
+                  "host": ["{{baseUrl}}"],
+                  "path": ["core", "v1", "openid", "configuration"]
+                }
+              },
+              "response": []
+            }
+          ]
+        },
+        {
+          "name": "User",
+          "item": [
+            {
+              "name": "Search Users",
```

Comment:
should this be "Create user" ?

Reply:
I don't think it should.

There's no `create-user` endpoint on api-ref. Plus, we don't want devs to manually add `users` onto their tenant, they should've been synced by us every time a new user authenticates via oid4vci.